### PR TITLE
feat: ncc build for faster boot

### LINKED ISSUE #4: Add build to the Action to improve start time
The Telegram action could have a build to improve its latency.

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #4
QA: https://github.com/Meniole/telegram-bridge/actions/runs/11606855132/job/32319469343

### CODE REVIEW

**Comment by Keyrxng** [comment]
https://github.com/ubq-testing/telegram--bot/actions/runs/11611417342/job/32332706870 - ~10s creation
https://github.com/ubq-testing/telegram--bot/actions/runs/11611425483/job/32332732294 - ~10s close
https://github.com/ubq-testing/telegram--bot/actions/runs/11611436270/job/32332767135 - ~6s reopen

**Review by Keyrxng — APPROVED** [review]
QA is good, actions are faaast. I'm okay with the other minor changes. The `Error` casts I don't think are required but no reason to hold back merging as all error handling will be improved in a specific PR soon

File: tests/main.test.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe("Plugin tests", () => {
   });
 
   it("Should serve the manifest file", async () => {
-    const worker = require("../src/worker").default;
+    const { default: worker } = await import("../src/worker");
```
Comment (0x4007) [inline tests/main.test.ts:28]: Default is a hack?

**Review by 0x4007 — APPROVED** [review]

File: tests/main.test.ts (hunk 1 above)
Comment (gentlementlegen) [reply tests/main.test.ts:28]: What do you mean? Default is the default exported from the file. Like export default xxx

**Pull request merged**